src/bin/www.ts

```diff
@@ -0,0 +1,20 @@
+import Koa = require('koa');
+import {productsRouter} from '../routers/product-router';
+import bodyParser = require('koa-bodyparser');
+import {loggingMiddleware} from '../middlewares/logging-middleware';
+import {config} from '../../configurations/configuration';
+import {errorHandlerMiddleware} from '../middlewares/error-handler';
+import {connectToDB} from '../actions/db-connector';
+
+
+const app = new Koa();
+connectToDB();
```

Comment:
dont listen to app untill you connected to the DB. what you'r server can do without DB?

Reply:
done